prog/probe.go

```diff
@@ -221,3 +233,28 @@ func getFirstAddressOf(name string) (string, error) {
 		return "", fmt.Errorf("No address found for %s", name)
 	}
 }
+
+// TODO(paulbellamy): where should this live?
```

Comment:
I wanted to keep the `plugins` package fairly separate from the concerns of scope, with an eye to extracting it into a separate package at some point.